sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -229,8 +241,35 @@ public boolean isSampled() {
     }
 
     @Override
-    public Map<String, AttributeValue> attributes() {
+    public Map<String, AttributeValue> getAttributes() {
       return Collections.emptyMap();
     }
   }
+
+  /** Probability-based sampling decision with a single attribute for the probability. */
+  @Immutable
+  @AutoValue
+  abstract static class ProbabilityDecision implements Decision {
+
+    ProbabilityDecision() {}
+
+    /**
+     * Creates sampling decision without attributes.
+     *
+     * @param decision sampling decision
+     * @param probability the probability that was used for the decision.
+     */
+    static ProbabilityDecision create(boolean decision, double probability) {
+      return new AutoValue_Samplers_ProbabilityDecision(
+          decision,
+          singletonMap(
```

Comment:
nit: I like more to see `Collections.singletonMap` :)

Reply:
really? for methods that are really clear in their naming (as opposed to something like `ImmutableMap.of(...)`), I very much prefer the static import version of it.

Reply:
Personal preference, no need to do it :)